fido/update_signatures.py

```diff
@@ -94,7 +94,7 @@ def run(defaults=defaults):
             print("Creating temporary folder for download:", tmpdir)
             try:
                 os.mkdir(tmpdir)
-            except:
+            except:  # noqa: E722
```

Comment:
Looking up the docs, it looks like the only exception that can occur here is [`OSError` (2.7)](https://docs.python.org/2.7/library/os.html#os.mkdir) and [`FileExistsError`/`PermissionError` (3)](https://docs.python.org/3/library/os.html#os.mkdir) - what about catching the specific exception?

Reply:
Sounds good. Hey it looks like `FileExistsError` and `PermissionError` are subclasses of `OSError`, that's nice. I thought for a moment that I was going to have to make some weird aliases hehe :)

Reply:
Oh, good! I was worried about that for a minute, haha.